lib/ansible/playbook/play_context.py

```diff
@@ -242,11 +242,9 @@ class PlayContext(Base):
     _verbosity = FieldAttribute(isa='int', default=0)
     _only_tags = FieldAttribute(isa='set', default=set())
     _skip_tags = FieldAttribute(isa='set', default=set())
-    _check_mode = FieldAttribute(isa='bool', default=False)
```

Comment:
yes, was redundant and no, as we check for None in task, which means its 'not set' vs True/False which would override the config and cli options